src/__tests__/App.test.jsx

```diff
@@ -1,8 +1,20 @@
 import App from '../App.jsx'
-import { describe, it, expect } from 'vitest';
+import { describe, it, beforeEach, afterEach, expect, vi } from 'vitest';
 import { render, fireEvent, screen } from "@testing-library/react";
+// Extra tests:
+// bday must be valid date - day can't be above 31/30/29 depending on month
```

Comment:
If you write // TODO a lot of IDEs will give you a reminder that this needs to be done later, rather than a comment like this getting left behind